docs/source/notebooks/time_series_analysis.ipynb

```diff
@@ -31,11 +31,8 @@
     "import os\n",
     "import xarray as xr\n",
     "\n",
-    "# Disable Dataset.__repr_html_ and DataArray._repr_html_, enabled by default in xarray 0.15.1\n",
-    "xr.set_options(display_style='text')\n",
-    "\n",
     "# Set environment variable WPS_URL to \"http://localhost:9099\" to run on the default local server\n",
-    "url = os.environ.get(\"WPS_URL\", \"https://pavics.ouranos.ca/twitcher/ows/proxy/raven/wps\")\n",
+    "url = os.environ.get(\"FINCH_WPS_URL\", \"https://pavics.ouranos.ca/twitcher/ows/proxy/finch/wps\")\n",
```

Comment:
I assume this is the feature that migrated to Finch?  Now an existential question, should this notebook be moved to Finch instead? :D

Reply:
Conceptually, it's part of Raven use cases (and it uses a wps process from raven at the very end).